README.md

```diff
@@ -1,8 +1,13 @@
 # falcon-integration-gateway [![Python Lint](https://github.com/CrowdStrike/falcon-integration-gateway/actions/workflows/linting.yml/badge.svg)](https://github.com/CrowdStrike/falcon-integration-gateway/actions/workflows/linting.yml) [![Container Build on Quay](https://quay.io/repository/crowdstrike/falcon-integration-gateway/status "Docker Repository on Quay")](https://quay.io/repository/crowdstrike/falcon-integration-gateway)
 
-Falcon Integration Gateway forwards findings from CrowdStrike Falcon platform to various [backends](fig/backends).
+Falcon Integration Gateway (FIG) forwards threat detection findings from CrowdStrike Falcon platform to the [backend](fig/backends) of your choice.
 
-Status: This is pre-release version (WIP).
+Learn more about CrowdStrike Falcon Threat Graph: [Product Page](https://www.crowdstrike.com/products/falcon-platform/threat-graph/), [Data Sheet](https://www.crowdstrike.com/resources/data-sheets/threat-graph/).
+
```

Comment:
What's the correlation between FIG forwarding threat detection findings and reading more about the threat graph?

Reply:
Nod, not clear.

Do we have better document we can point to with regards to Falcon detection capabilities?

Reply:
Hmm. Would it be accurate to say the "... the FIG forwards events from CrowdStrike Falcon blah blah. The full event dictionary can be found at <<URL>>."

And link to the event dictionary? This way we define what gets forward and provide link for additional description/details.

Reply:
@shawndwells, thanks. I rewrote the introductory text. Can you please take a second look?